app_dart/test/request_handlers/get_timeseries_history_test.dart

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_service/src/model/appengine/commit.dart';
+import 'package:cocoon_service/src/model/appengine/time_series.dart';
+import 'package:cocoon_service/src/model/appengine/time_series_value.dart';
+import 'package:cocoon_service/src/request_handlers/get_timeseries_history.dart';
+import 'package:cocoon_service/src/service/datastore.dart';
+import 'package:test/test.dart';
+
+import '../src/datastore/fake_cocoon_config.dart';
+import '../src/datastore/fake_datastore.dart';
+import '../src/request_handling/regular_request_handler_tester.dart';
+
+void main() {
+  group('GetTimeSeriesHistory', () {
+    FakeConfig config;
+    FakeDatastoreDB db;
+    GetTimeSeriesHistory handler;
+    RegularRequestHandlerTester tester;
+
+    setUp(() {
+      config = FakeConfig();
+      db = FakeDatastoreDB();
+      tester = RegularRequestHandlerTester();
+      tester.requestData = <String, dynamic>{
+        'TimeSeriesKey':
+            'ahNzfmZsdXR0ZXItZGFzaGJvYXJkcjULEgpUaW1lc2VyaWVzIiVhbmFseXplcl9iZW5jaG1hcmsuZmx1dHRlcl9yZXBvX2JhdGNoDA',
```

Comment:
Use a shorter string?

Reply:
This is a real key, and we need to test that.

Reply:
Why? You are talking to a fake datastore, you can use a fake key.

Reply:
This key is url safe, and it is visible to every one even you are not logged in.
This key is exactly the one our API uses
In the API, we need to decode that key to the real timeseries, which is also pre loaded in our test. Together with the above two facts, it does not hurt to use this real key:-)